anylabeling/services/auto_labeling/yolov5_seg.py

```diff
@@ -0,0 +1,223 @@
+import logging
+import os
+
+import cv2
+import numpy as np
+from PyQt5 import QtCore
+
+from anylabeling.views.labeling.shape import Shape
+from anylabeling.views.labeling.utils.opencv import qt_img_to_cv_img
+
+from .model import Model
+from .types import AutoLabelingResult
+
+
+class YOLOv5_seg(Model):
```

Comment:
Please rename `YOLOv5_seg` to `YOLOv5Segmentation` to follow [PEP8](https://peps.python.org/pep-0008/#class-names).

Reply:
Done